[REF, DOC] Document PAID combination method (#264)
ME-ICA/tedana: docs/approach.rst

.. image:: /_static/10_optimal_combination_timeseries.png
  :align: center

> .. note::

Reviewer: Can we have each sentence on a newline ? Sorry !

Author: Done.

>     The tedana package also supports another method for optimal combination that

Reviewer: ```suggestion
    An alternative method for optimal combination that
```

>     does not use :math:`T_{2}^*`, known as the parallel-acquired inhomogeneity

Reviewer: ```suggestion
    does not use :math:`T_{2}^*`, is the parallel-acquired inhomogeneity
```

    desensitized (PAID) ME-fMRI combination method (`Poser et al., 2006`_).

>     This method is not supported in the tedana workflow, however, because

Reviewer: Since the t2smap workflow is called as part of the tedana workflow, this is a little confusing, to me.

Author: We could hypothetically allow users to run tedana without denoising (e.g., `--disable-denoising`?), which would be the same as the t2smap workflow, but that would mean managing how the combmode and denoising arguments interact (i.e., using paid and denoising should raise a warning and switch paid to t2s automatically).

Reviewer: Right, sorry, I wasn't thinking quite like that ! I just meant that it is technically in the tedana workflow, but it's not accessible, you're right.

Maybe we could just say something like 
```
This method specifically assumes that noise in the acquired echoes is "isotopic and 
homogeneous throughout the image," meaning it should be used on smoothed data. 
As we do not recommend performing tedana denoising  on smoothed data, 
we discourage using PAID within the tedana workflow.
We do, however, make it accessible as an alternative combination method 
in the t2smap workflow.
```
